app/controllers/appropriate_bodies/appropriate_bodies_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppropriateBodies
+  class AppropriateBodiesController < BaseController
+    layout "application"
+
+    def index
+      @choose_organisation_form = ChooseOrganisationForm.new
+      @choose_organisation_form.user = current_user
+
+      if @choose_organisation_form.only_one?
+        redirect_to appropriate_body_participants_path(@choose_organisation_form.appropriate_body)
+      end
+    end
+
+    def create
+      @choose_organisation_form = ChooseOrganisationForm.new(choose_organisation_form_params)
+      @choose_organisation_form.user = current_user
+
+      if @choose_organisation_form.valid?
```

Comment:
Should it be saving here in addition to checking it's valid?

Reply:
@peteryates we don't need to save here, we only want to redirect to the correct organisation (appropriate body).

Reply:
Ah - fair enough, thanks @mooktakim. It being in `#create` threw me 😅